parity/blockchain.rs

```diff
@@ -98,6 +98,7 @@ pub struct ImportBlockchain {
 	pub with_color: bool,
 	pub verifier_settings: VerifierSettings,
 	pub light: bool,
+	pub max_round_blocks_to_import: usize,
```

Comment:
We probably don't need to add this field here and just use the default (12). Same below for `Export*` structs.

Reply:
It is a valid option indeed, I just assumed it wouldn't make much of a difference without networking. I'm OK with with keeping it configurable.